site/src/components/DropdownButton/DropdownButton.stories.tsx

```diff
@@ -23,7 +22,7 @@ WithDropdown.args = {
 
 export const WithCancel = Template.bind({})
 WithCancel.args = {
-  primaryAction: <DisabledButton workspaceState={WorkspaceStateEnum.deleting} />,
+  primaryAction: <DisabledButton workspaceStatus="deleting" />,
```

Comment:
Curious why we're no longer using the enum here?

Reply:
The WorkspaceStatus type is defined by the backend now